netbox/extras/webhooks_worker.py

```diff
@@ -0,0 +1,52 @@
+import requests
+import hmac
+import hashlib
+from rq.utils import import_attribute
+
+from django_rq import job
+
+from extras.constants import WEBHOOK_CT_JSON, WEBHOOK_CT_X_WWW_FORM_ENCODED
+
+
+@job('default')
+def process_webhook(webhook, data, model_class, event, signal_received_timestamp):
+    """
+    Make a POST request to the defined Webhook
+    """
+    payload = {
+        'event': event,
+        'signal_received_timestamp': signal_received_timestamp,
```

Comment:
Can we just call this `timestamp`? I realize the signal might not fire at the exact moment an object is saved/deleted, but I think people will assume this is the webhook firing time anyway.